packages/components/src/templates/button-with-badge.stories.ts

```diff
@@ -0,0 +1,84 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Button with Badge',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/hER2N8wZXhiTrdlDXyrdUt/Quote?type=design&node-id=1001-4293&mode=design&t=Xywhix1rQMoatokH-0'
+    }
+  }
+};
+/**
+ * Here are some examples of sd-button working with sd-badge.
+ */
+export const Default = {
+  render: () => html`
+    <section class="flex p-4 gap-10 items-center">
```

Comment:
Pls see [figma template](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2501-4280&t=8XG4ATjdx0TEu0Dm-11)

And adjust what we show accordingly.

Inverted can be just added directly underneath. No extra headline/subline neccessary.